### ISSUE TITLE
My browser does not allow the self certified SSL for localhost

### ISSUE THREAD

**Issue by sonicjoy** [dotnet/core#8956]
Problem encountered on https://dotnet.microsoft.com/en-us/learn/aspnet/blazor-tutorial/run
Operating System: windows

Provide details about the problem you're experiencing. Include your operating system version, exact error message, code sample, and anything else that is relevant.

**Comment by rzikm** [comment]
Have you checked the information at the bottom of the page? Sometimes the browser needs to be restarted to pick up the dev certificate. If that still does not work, can you try running?

```
dotnet dev-certs https --trust
```

**Comment by sonicjoy** [comment]
Hi, yes I installed the certificate and trusted the entity, the issue seems to be with Edge browser. When I use Chrome to run it, there is no such issue.

**Comment by rzikm** [comment]
I tried it and there should be an option to ignore the error in Edge, see the link at bottom-left
<img width="552" alt="image" src="https://github.com/dotnet/core/assets/32671551/9fb25574-9c6b-410d-8d60-a1308ccf1600">

**Comment by sonicjoy** [comment]
Yeah, usually that page was what I'd get, but not after I installed
the certificate.

I just tried it again, and it worked without any warnings. I guess
restarting my machine might have reloaded the certificate for Edge? Thanks
for your help!
